k8s/demo/mongodb/README.md

```diff
@@ -0,0 +1,453 @@
+# Running mongodb statefulset on OpenEBS 
+
+This tutorial provides instructions on how to run a mongodb statefulset on OpenEBS storage in a kubernetes cluster. 
+It also provides steps to generate a standard OLTP load on the database using a custom sysbench tool and test the data replication 
+across the mongodb instances.
+
+## Pre-requisites
+
+Pre-requisites include the following:
+
+- A fully configured kubernetes cluster (versions 1.6.3/4/6 & 1.7.0 have been tested) with K8S master and at least one K8S node. 
+This maybe created on cloud platforms like GKE, on-premise virtual machines (vagrant/VMware/Hyper-V) or bare-metal boxes
+
+  Note:
+  
+  - It is recommended to use a 3-node cluster, with one master and two nodes, as this will aid creation of storage replicas on separate 
+nodes. This is especially helpful to maintain redundancy and data availability.
+
+    ```
+    karthik@MayaMaster:~$ kubectl get nodes
+    NAME         STATUS    AGE       VERSION
+    mayahost01   Ready     18h       v1.6.3
+    mayahost02   Ready     18h       v1.6.3
+    mayamaster   Ready     18h       v1.6.3
+    ```
+    
+- Sufficient resources on the minions to host the openebs storage pods & application pods. This includes sufficient disk space, 
+  as, in this example, physical storage for the volume containers shall be carved out from the local storage
+  
+- iSCSI support on the minions. This is needed to be able to consume the iSCSI target exposed by the openebs volume container (i.e., VSM). 
```

Comment:
minions = nodes
needed to be able to = required to consume
openebs = OpenEBS
VSM we are not using now right?